MARKET_ADD.md

```diff
@@ -0,0 +1,88 @@
+        "symbol": "mWETH",
+        "addressesString": "MOONWELL_WETH",
+        "jrm": {
+            "baseRatePerYear": 0.02e18,
+            "multiplierPerYear": 0.15e18,
+            "jumpMultiplierPerYear": 3e18,
+            "kink": 0.6e18
+        }
+```
+- `initialMintAmount` amount of tokens to mint during gov proposal, if token has 18 decimals, should be `1e12`, if token has 6 decimals, should be `1e6`. If token has greater than 18 decimals, decimals should be token decimals - 6 decimals. Temporal Governor must be funded with the initialMintAmount of said tokens before the proposal is executed in order for the proposal to succeed.
+- `collateralFactor` the percentage of value of supplied assets that is counted towards an account's collateral value, scaled up by `1e18`. Must be less than `1e18` to ensure system solvency.
+- `reserveFactor` percentage of interest accrued that goes to the Moonwell DAO reserves, scaled up by `1e18`.
+- `seizeShare` percentage of seized collateral that goes to protocol reserves when a liquidation occurs, scaled up by `1e18`.
+- `supplyCap` cap of amount of assets that can be supplied for a given market.
+- `borrowCap` cap of amount of assets that can be borrowed for a given market.
+- `priceFeedName` name of the chainlink aggregator address in `Addresses.sol`. **Note user must add the address to Addresses.sol on the proper network pre-deployment in order for the price feed to be set correctly**
+- `tokenAddressName` name of the underlying token address name for the new market, set in `Addresses.sol`. **Note user must add the underlying token address to Addresses.sol on the proper network pre-deployment in order for the MToken's underlying address to be set correctly**
+- `name` name of the Moonwell MToken. Prefixed with `Moonwell`.
+- `symbol` symbol of the Moonwell MToken. Prefixed with `m`.
+- `addressesString` name of the address string set in `Addresses.sol`.
+- `jrm` parameters for the configuration of a JumpRateModel for each MToken.
+- `baseRatePerYear` cost to borrow per year as a percentage, scaled by `1e18`.
+- `multiplierPerYear` multiplier on the base rate, which creates the curve of the rate before kink is hit, as a percentage, scaled by `1e18`.
+- `jumpMultiplierPerYear` rate multiplier as a percentage, scaled by `1e18` after kink is hit.
+- `kink` the point on the utilization curve after which the interest rate spikes using `jumpMultiplierPerYear` as a percentage, scaled by `1e18`
+
+## 2. Proposal Description
+
+Once the proposal description has been created, copy and paste it into [`MarketListingDescription.md`](./test/proposals/mips/examples/mip-market-listing/MarketListingDescription.md), deleting all previous data from this file.
+
+## 3. Environment Variables
+Once both the `MarketListingDescription.md` and `MTokens.json` file have the necessary contents, environment variables must be set for the script to read in their path. In the example folder, run these commands:
```

Comment:
Is it supposed to be run in the example folder, or in the root folder of the repository? The export commands shouldn't care where they are executed, but the `forge script...` command below definitely does care.

Reply:
this command should be run in the root of a project

Reply:
it looks like when you're running the tests it doesn't care what folder you're in, it reads in the env vars properly